### ISSUE TITLE
Regession, from version 5, with 6.0.0-beta2: hang when launching a program via a 'shortcut'

### ISSUE THREAD

**Issue by LinuxOnTheDesktop** [Ulauncher/Ulauncher#1296]
### Environment

Linux Mint 21.2 Cinnamon

### Bug description

When I launch this 'shortcut'

![image](https://github.com/Ulauncher/Ulauncher/assets/47056543/e409e2c0-90d6-4028-9a89-a4b6441352c4)

. . I see the following (perhaps only after trying to interact with the Ulauncher window)

![image](https://github.com/Ulauncher/Ulauncher/assets/47056543/bb370609-2d0c-48ec-9b5c-c34ad486166d)

and choosing 'quit' quits the application that I launched as well as Ulauncher itself (though Ulauncher's tray icon remains and remains functional). (I find that your 'Quirks and workaround' document says, in effect, that, on my system, Ulauncher should not take down with it an application that it has launched. For, my system is Mint 21.2, which is based upon Ubuntu 22.04.)

The 'shortcut' at issue is one of Ulauncher's own shortcuts. The program to which the shortcut points works fine when I launch it directly from Ulauncher. (So why did I have the shortcut? I forget.) Other shortcuts that I have seem to work. I am afraid that, after experiencing the problem (twice), and in an attempt at troubleshooting, I deleted the problematic shortcut.

I believe that I did not have the problem with Ulauncher version 5. (I did not try beta1 of version 6).

The log below is from a session in which - I think - I had the problem.

### Log output

```shell
2023-11-01 05:36:19,349 | INFO | Ulauncher version 6.0.0-beta2 | main.main():63
2023-11-01 05:36:19,349 | INFO | Extension API version 3.0 | main.main():64
2023-11-01 05:36:19,349 | INFO | GTK+ 3.24.33 | main.main():65
2023-11-01 05:36:19,349 | INFO | PyGObject+ 3.42.1 | main.main():66
2023-11-01 05:36:19,349 | INFO | Desktop: X-CINNAMON (X11) on Linux Mint 21.2 | main.main():68
2023-11-01 05:36:19,349 | WARNING | 

╔═════════════════════════════════════════════════════════════════════════════╗
║                  YOU ARE RUNNING A PRE-RELEASE of ULAUNCHER.                ║
║ Please do not report extension API support warnings to extension developers ║
║ We are still in the process of developing and documenting these features    ║
╚═════════════════════════════════════════════════════════════════════════════╝

 | main.main():70
2023-11-01 05:36:19,350 | INFO | Migrating /home/<myUserName - redacted>/.config/ulauncher/ext_preferences/com.github.psukys.ulauncher-plugin-windows.json to /home/<myUserName - redacted>/.config/ulauncher/ext_preferences/com.github.psukys.ulauncher-plugin-windows.json | migrate._migrate_file():44
2023-11-01 05:36:19,350 | INFO | Migrating /home/<myUserName - redacted>/.config/ulauncher/ext_preferences/com.github.fisadev.ulauncher-better-file-browser.json to /home/<myUserName - redacted>/.config/ulauncher/ext_preferences/com.github.fisadev.ulauncher-better-file-browser.json | migrate._migrate_file():44
2023-11-01 05:36:19,350 | INFO | Migrating /home/<myUserName - redacted>/.config/ulauncher/ext_preferences/com.github.mikebarkmin.ulauncher-duckduckgo.json to /home/<myUserName - redacted>/.config/ulauncher/ext_preferences/com.github.mikebarkmin.ulauncher-duckduckgo.json | migrate._migrate_file():44
2023-11-01 05:36:19,350 | INFO | Migrating /home/<myUserName - redacted>/.config/ulauncher/ext_preferences/com.github.episode6.ulauncher-system-management-direct.json to /home/<myUserName - redacted>/.config/ulauncher/ext_preferences/com.github.episode6.ulauncher-system-management-direct.json | migrate._migrate_file():44
2023-11-01 05:36:19,351 | INFO | Migrating /home/<myUserName - redacted>/.config/ulauncher/ext_preferences/com.github.brpaz.ulauncher-file-search.json to /home/<myUserName - redacted>/.config/ulauncher/ext_preferences/com.github.brpaz.ulauncher-file-search.json | migrate._migrate_file():44
2023-11-01 05:36:19,351 | INFO | Migrating /home/<myUserName - redacted>/.config/ulauncher/ext_preferences/com.github.iboyperson.ulauncher-system.json to /home/<myUserName - redacted>/.config/ulauncher/ext_preferences/com.github.iboyperson.ulauncher-system.json | migrate._migrate_file():44
2023-11-01 05:36:19,368 | INFO | Migrating /home/<myUserName - redacted>/.config/ulauncher/ext_preferences/com.github.dalanicolai.gnome-tracker-extension.json to /home/<myUserName - redacted>/.config/ulauncher/ext_preferences/com.github.dalanicolai.gnome-tracker-extension.json | migrate._migrate_file():44
2023-11-01 05:36:19,368 | INFO | Migrating /home/<myUserName - redacted>/.config/ulauncher/ext_preferences/com.github.dcervenkov.ulauncher-z-search.json to /home/<myUserName - redacted>/.config/ulauncher/ext_preferences/com.github.dcervenkov.ulauncher-z-search.json | migrate._migrate_file():44
2023-11-01 05:36:19,368 | INFO | Migrating /home/<myUserName - redacted>/.config/ulauncher/ext_preferences/com.github.dhelmr.ulauncher-duckduckgo-bangs.json to /home/<myUserName - redacted>/.config/ulauncher/ext_preferences/com.github.dhelmr.ulauncher-duckduckgo-bangs.json | migrate._migrate_file():44
2023-11-01 05:36:19,369 | INFO | Migrating /home/<myUserName - redacted>/.config/ulauncher/ext_preferences/com.github.leinardi.ulauncher-systemctl.json to /home/<myUserName - redacted>/.config/ulauncher/ext_preferences/com.github.leinardi.ulauncher-systemctl.json | migrate._migrate_file():44
```

### Communication guidelines

- [X] I have read and agree with Ulauncher's communication guidelines and checked the troubleshooting page for self-help.

### Not an extension issue

- [X] This issue applies to the Ulauncher application, not a specific extension.

### Ulauncher v6

- [X] I have made sure the bug isn't already fixed in v6.

**Comment by friday** [comment]
Can't really investigate this without the code from your shortcut.

**Comment by LinuxOnTheDesktop** [comment]
I think that the shortcut consisted merely of the command that, from the command-line, launches the relevant application. Thus, the shortcut was: `rclone-browser`; the relevant executable is /usr/local/bin/rclone-browser. And that application is [this one](https://kapitainsky.github.io/RcloneBrowser/), which is a QT application (and, as I wrote, runs fine when launched _directly_ from Ulauncher).

**Comment by friday** [comment]
Closing this as a duplicate of #1299 until you can reproduce this without pkexec.
